### PR TITLE
feat : created a Modal component and refactor links.

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                    
+                    .grid__item{
+                        background-color: rgb(159, 230, 159);
+                        padding: 4rem;
+                        font-size: large;
+                        text-align: center;
+                    } </code>
+                
+                  </pre>
+
```
Comment (Varun-28) [inline Grid/grid.html:195]: > Just use prettier extension, rest things are fine

File: Modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modal | Page</title>
+    <link rel="stylesheet" href="../Modal/modal.css">
+
+    
+
```
Comment (Jayesh3636) [inline Modal/modal.html:11]: remove unnecessary spaces

File: Modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+    <script src="https://kit.fontawesome.com/c9ffd32afb.js" crossorigin="anonymous"> </script>
+
+</head>
+<body>
+
+
+
+
```
Comment (Jayesh3636) [inline Modal/modal.html:27]: can also use beautify file extension to format the code

File: Modal/modal.html [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+            
+                    .modal-content {
+                        position: relative;
+                        display: flex;
+                        flex-direction: column;
+                        width: 100%;
+                        pointer-events: auto;
+                        background-color: #fff;
+                        background-clip: padding-box;
+                        border: 1px solid rgba(0,0,0,.2);
```
Comment (Jayesh3636) [inline Modal/modal.html:172]: try to use variables for colors

File: Modal/modal.html [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+                    .modal-header .btn-close {
+                        padding: 0.5rem 0.5rem;
+                        margin: -0.5rem -0.5rem -0.5rem auto;
+                    }
+                    .btn-close {
+                        box-sizing: content-box;
+                        width: 1em;
+                        height: 1em;
+                        padding: 0.25em 0.25em;
+                        color: #000;
```
Comment (Jayesh3636) [inline Modal/modal.html:207]: same here

File: Modal/modal.html [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+                        pointer-events: none;
+                    }
+                    .modal-footer {
+                        display: flex;
+                        flex-wrap: wrap;
+                        flex-shrink: 0;
+                        align-items: right;
+                        justify-content: flex-end;
+                        padding: 0.75rem;
+                        border-top: 1px solid #dee2e6;
```
Comment (Jayesh3636) [inline Modal/modal.html:225]: try to use single color notation eg either hex or rgba

**Review by Jayesh3636 — APPROVED** [review]
Just mentioned a few changes in the comments rest good to go

**Pull request merged**